### PR TITLE
add dbnet yaml for synthtext, mlt2017, td500, totaltext and ctw1500 dataset

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation
add dbnet yaml for synthtext, mlt2017, td500, totaltext and ctw1500 dataset
(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs
related issues #222 
(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]

**Review by HaoyangLee — APPROVED** [review]

File: configs/det/dbnet/README.md [hunk 1]
```diff
@@ -87,11 +88,41 @@ DBNet may generate inaccurate or discrete bounding boxes.
...
 |-------------------|----------------|--------------|----------------|------------|---------------|-------------|--------------|----------------|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| DBNet (ours)      | D910x1-MS2.0-G | ResNet-50    | SynthText       | 82.47%     | 87.75%        | 85.03%      | 13.3 s/epoch  | 51.1 img/s      | [yaml](db_r50_td500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_td500-0d12b5e8.ckpt)  |
+| DBNet            | D910x1-MS2.0-G | ResNet-18    | SynthText       | 79.21%     | 88.31%        | 83.51%      | 9.9 s/epoch  | 68.8 img/s      | [yaml](db_r18_td500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet18_td500-b5abff68.ckpt)  |
```
Comment (SamitHuang) [inline configs/det/dbnet/README.md:91]: SynthText  补超链接到pretrained weight下载地址。

**Review by HaoyangLee — APPROVED** [review]

**Review by HaoyangLee — APPROVED** [review]

File: configs/det/dbnet/db_r50_mlt2017.yaml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  head:
+    name: DBHead
+    k: 50
+    bias: False
+    adaptive: True
+  pretrained: https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_synthtext-40655acb.ckpt
+
+postprocess:
+  name: DBPostprocess
+  output_polygon: True   # whether to output a polygon or a box
```
Comment (hadipash) [inline configs/det/dbnet/db_r50_mlt2017.yaml:29]: MLT2017 has boxes as labels, isn't it?

File: configs/det/dbnet/db_r50_totaltext.yaml [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  loader:
+    shuffle: True
+    batch_size: 20
+    drop_remainder: True
+    num_workers: 20
+
+eval:
+  ckpt_load_path: tmp_det/best.ckpt
+  dataset_sink_mode: False
+  ema: True
```
Comment (hadipash) [inline configs/det/dbnet/db_r50_totaltext.yaml:126]: `ema` should be inside `train`?

Comment (Songyuanwei) [reply configs/det/dbnet/db_r50_totaltext.yaml]: fixed

File: configs/det/dbnet/db_r50_mlt2017.yaml (hunk 2 above)
Comment (Songyuanwei) [reply configs/det/dbnet/db_r50_mlt2017.yaml]: fixed

File: configs/det/dbnet/README.md (hunk 1 above)
Comment (Songyuanwei) [reply configs/det/dbnet/README.md]: fixed

**Review by HaoyangLee — APPROVED** [review]

File: configs/det/dbnet/README.md [hunk 4]
```diff
@@ -76,11 +70,8 @@ DBNet may generate inaccurate or discrete bounding boxes.
...
-=======
-| DBNet++             | D910x1-MS2.0-G | ResNet-50     | SynthText  | 85.70%     | 87.81%        | 86.74%      | 17.7 s/epoch | 56 img/s  | [yaml](db++_r50_icdar15.yaml)       | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50-068166c2.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50-068166c2-76fcb451.mindir)   |
+| DBNet++             | D910x1-MS2.0-G | ResNet-50     | [SynthText](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_synthtext-40655acb.ckpt)  | 85.70%     | 87.81%        | 86.74%      | 17.7 s/epoch | 56 img/s  | [yaml](db++_r50_icdar15.yaml)       | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50-068166c2.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnetpp_resnet50-068166c2-76fcb451.mindir)   |
```
Comment (SamitHuang) [inline configs/det/dbnet/README.md:74]: 预训练模型没必要提供mindir, 推理不用。

**Review by SamitHuang — APPROVED** [review]

**Pull request merged**